pyquil/api/_compiler.py

```diff
@@ -354,5 +349,4 @@ def reset(self):
         """
         Reset the state of the QPUCompiler Client connections.
         """
-        pyquil_config = PyquilConfig()
-        self.client = refresh_client(self.client, pyquil_config.quilc_url)
+        self._qpu_compiler_client = None
```

Comment:
Do you want to refresh `self.client` here rather than `self._qpu_compiler_client`? Similar to how `quilc_client` is handled for the `QPUCompiler` above.

Reply:
Good catch!